README.md

```diff
@@ -14,14 +14,10 @@ the process of aligning security controls from SRG items into actual STIG securi
 * Confidential data in the database is encrypted using symmetric encryption
 * Authenticate via the local server, through github, and through configuring an LDAP server.
 
-## Installation
-
-### Dependencies
-
-For Docker:
-  * Docker
-  * docker-compose
+## Configuration Breakdown
+[Configuration](docs/config.md)
 
+## Dependencies:
```

Comment:
Need to make this more clear that it is "Development Dependencies" and not just Dependencies for users trying to Deploy Vulcan.